helper/diffPlaces.js

```diff
@@ -174,6 +223,7 @@ function isAddressDifferent(item1, item2){
  * Optionally provide $requestLanguage (req.clean.lang.iso6393) to improve name deduplication.
  */
 function isDifferent(item1, item2, requestLanguage){
+  if( isEquivalentIdentity( item1, item2 ) ){ return false; }
```

Comment:
Might be worth pointing out here that `isEquivalentIdentity` is different (no pun intended) than all the other checks in the `isDifferent` function.

All the others are each sufficient to prove that items are different. `isEquivalentIdentity`, on the other hand, is sufficient to prove that the items are the _same_.

I'm honestly surprised it took us this long to have such a check but we might want to divide the functions within `isDifferent` into two appropriate sections with a comment or something to help us keep track of them.